style.css

```diff
@@ -0,0 +1,205 @@
+.Index
+body {
+    background-image: url(/images/spacebg.jpeg); 
+    background-size: 120%;
+    background-repeat: no-repeat;
+    color: thistle;
+    text-align:center;
+    }
+
+.Dianaport {
+    width: 200px;
+    float: left;
+}
+
+.Dianabout {
+    width: 200px;
+    float:right;
+}
+.about-header {
+    font-size: 50px;
+}
+.about-img {
+    width: 30%;
+}
+
+.about-D {
+    background-color:peru;
+    text-align: center;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.Dianas-paragraph {
+    color: black;
+    text-align:left;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.Portfolio {
+    background-color: cornflowerblue;
+    text-align: center;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size:40px;
+    color:black;
+}
+
+.portfolio-text {
+    font-size: 15PX;
+    margin-left: auto;
+    margin-right: auto;
+    height: 70%;
+    width: 70%;
+    opacity: 100;
```

Comment:
I'm pretty sure this needs to be a decimal number like `0.87`.  I think it should be 0 <= x <= 1